hadoop-hdds/docs/content/concept/StorageContainerManager.zh.md

```diff
@@ -4,7 +4,7 @@ date: "2017-09-14"
 weight: 3
 menu: 
   main:
-     parent: 概念
+     parent: 架构
```

Comment:
The rename of the parent menu I think can be handled in the new issue because it also contains other items.

![image](https://user-images.githubusercontent.com/14295594/98642466-7ddae300-2368-11eb-8bd2-31fa0f665446.png)

Inconsistent changes will cause some menus to disappear.